web/setup.py

```
@@ -163,6 +163,157 @@ def get_role(role: str):
+        created_count = 0
+        skipped_count = 0
+        error_count = 0
+
+        app = create_app(config.APP_NAME + '-cli')
+        with (app.test_request_context()):
+            for user_entry in users_data:
+                try:
+                    # Validate required fields
+                    if 'username' not in user_entry and\
+                            'email' not in user_entry:
+                        print(f"Skipping user: missing 'username' or 'email'")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)